python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml

```diff
@@ -3,8 +3,6 @@
 
 ---
 nick: {{ cookiecutter.repo_name }}
-tags:
-    - tools
```

Comment:
Should this be "library", or is this used for multiple purposes? See https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0002-bp-repo-metadata.html#tags

Reply:
`python-template` is used as the base for multiple cookiecutters, so I don't think we can prefill the tags with anything.